packages/graph-retriever/src/graph_retriever/adapters/base.py

```diff
@@ -57,14 +53,7 @@ def search_with_embedding(
         contents :
             List of up to `k` content items most similar to the query vector.
         """
-        query_embedding = self.embed_query(query)
```

Comment:
Move search (without an emdedding) into the langchain case, along with `embed_query`. This makes it harder to (incorrectly) call `embed_query` within the traversal. We were only using it in tests, because the traversal path got the embedding along with the first batch of results.